src/main/jte/bookings/single.jte

```diff
@@ -45,7 +54,7 @@
     </p>
 @elseif (allowEditing)
     <br>
-    <label class="btn btn-ghost text-gray-500 text-sm soli-view-content" for="soli-edit-toggle">
+    <label class="btn btn-ghost text-gray-500 text-sm soli-view-content" for="soli-edit-toggle" aria-label="${context.localize("bookings.description.new")}" title="${context.localize("bookings.description.new")}">
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.